### DISCUSSION TITLE
This should have really good performance

### DISCUSSION THREAD

**Discussion by TheMowgliMan** [TheMowgliMan/Country-Clash#7] [Announcements]
On my Dad's PC (intel i5-8400k, nvidia gtx 1080), it gets a maximum of 2240 FPS right now, which I suspect is a cap.

**Comment by 3quarterscolon** [comment]
That’s really good. Let’s hope it stays that way, though.

**Reply by TheMowgliMan** [reply]
It won't, due to bot countries, automatic upkeep, etc, but if it drops below ~100 FPS I'd be surprised. This isn't even new hardware; It's almost three generations old.

**Reply by TheMowgliMan** [reply]
It uses multithreading to prevent FPS hits too.

**Comment by TheMowgliMan** [comment]
I capped it at 400 fps to prevent artifacts.

**Comment by ghost** [comment]
Nice! To me, anything faster than the 30 FPS I can get on Scratch using Firefox is excellent.

**Reply by TheMowgliMan** [reply]
Great! It runs >100fps on a 2014 HP Pavilion too, so I think we're all set. It uses threads to keep FPS high on the main thread, even if it is really running very slow.

**Reply by TheMowgliMan** [reply]
For the record, the minimum FPS it's supposed to get on this (The HP) system is 30, per the requirements, so we're all set.
